build/build-image/cross/Dockerfile

```diff
@@ -71,7 +71,7 @@ RUN go get golang.org/x/tools/cmd/cover \
             golang.org/x/tools/cmd/goimports
 
 # Download and symlink etcd. We need this for our integration tests.
-RUN export ETCD_VERSION=v3.1.10; \
+RUN export ETCD_VERSION=v3.2.14; \
```

Comment:
we probably need to bump the crossbuild image for this to have an effect in the integration tests.

Reply:
Thanks. I've updated `build/build-image/cross/VERSION` pushed `1.9.2-2`.